docs/how-to-git.md

```diff
@@ -1,25 +1,78 @@
-Second exercise was to properly understand how to push something to GitHub
+# How to Git
 
-```
-Clone repo:
-    -   git clone {git@github.com:Nitro... ...app.git}
+- [Architecture](#git-commands-architecture)
+- [Git commands](#git-commands)
+
+### Git Commands Architecture
 
-Create local branch:
-    -   git fetch origin
-    -   git checkout -b {branch-name} origin/main
+```mermaid
+sequenceDiagram
+    participant U as Working Directory (Untracked)
```

Comment:
`participant directory as Working Directory (Untracked)`

it will be easy to read the diagram if you use the proper names